ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -638,6 +639,14 @@ public class RouterConfig {
   @Config(ROUTER_UPDATE_OP_METADATA_RELIANCE_TIMESTAMP_IN_MS)
   public final long routerUpdateOpMetadataRelianceTimestampInMs;
 
+  /**
+   * If this config is set to {@code true}, when operation fails because of not enough replicas having replicated the Blob,
+   * the operation will trigger the on-demand replication to replicate the Blob to more replicas and then retry the request.
+   */
+  @Config(ROUTER_REPAIR_WITH_REPLICATE_BLOB_ENABLED)
+  @Default("false")
```

Comment:
You can remove the @Default annotation.  It is not used at all.  It only causes confusion.